website/docs/getting-started/02-prerequisites.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Prerequisites
+---
+
+This section details the prerequisites for deploying an AKS cluster with support for high-speed InfiniBand networking and Remote Direct Memory Access (RDMA), including optional configurations for GPUDirect RDMA.
+
+## AKS Nodepools
+
+An active AKS cluster is required as the foundation for deploying RDMA and InfiniBand capabilities. The cluster serves as the Kubernetes environment where Network Operator and GPU Operator (if using GPUDirect RDMA) will be installed.
+
+- **Requirement**: Create an AKS cluster using the [Azure Portal](https://portal.azure.com) or [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest). Ensure the cluster is running a supported Kubernetes version compatible with [Network Operator](https://docs.nvidia.com/networking/display/kubernetes2501/platform-support.html) and [GPU Operator](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/platform-support.html).
```

Comment:
```
Ensure the cluster is running a supported Kubernetes version compatible with
```

I think we have to ensure that the latest version is compatible with the supported AKS version instead of asking users to figure that part out.

Reply:
Since Network Operator and GPU Operator aren’t managed AKS add-ons, we can’t prevent users from installing outdated versions. They’re responsible for ensuring compatibility, so we recommend they follow this guide, install the latest versions, and use the suggested configurations.